.circleci/Dockerfile

```diff
@@ -11,3 +11,22 @@ RUN apt-get update && apt-get install -y \
   libxtst6 \
   rpm \
   xvfb
+
+# For puppeteer.
+# see https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker
+
+# See https://crbug.com/795759
+RUN apt-get update && apt-get install -yq libgconf-2-4
+
+# Install latest chrome dev package.
+# Note: this installs the necessary libs to make the bundled version of Chromium that Puppeteer
+# installs, work.
+RUN apt-get update && apt-get install -y wget --no-install-recommends \
+    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install -y google-chrome-unstable \
+      --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get purge --auto-remove -y curl \
+    && rm -rf /src/*.deb
```

Comment:
Why we’re touching `apt` so many times? Docker building will be faster and cleaner if you’d simplify it to the outline:

(1) Add puppeteer repo:

```dockerfile
RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
&& sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
```

(2) Update and install everyting

```dockerfile
RUN apt-get update && apt-get install --yes --no-install-recommends \
  google-chrome-unstable \
  wget \
  and \
  all \
  the \
  stuff \
  used to be here before
```

(3) [optional] cleanup